sky/job/dashboard/templates/index.html

```diff
@@ -107,6 +106,32 @@ <h1>Managed spot jobs</h1>
             </tbody>
         </table>
     </div>
+    <script>
+        document.querySelectorAll('.folder').forEach(function (folder) {
```

Comment:
Add comment: is folding for pipelines only?

Reply:
Added. Thanks!